src/options.js

```diff
@@ -55,7 +60,7 @@ export default {
       name: 'facebook',
       url: '/auth/facebook',
       authorizationEndpoint: 'https://www.facebook.com/v2.5/dialog/oauth',
-      redirectUri: typeof(window) === 'undefined' ? '/' : window.location.origin + '/',
+      redirectUri: getCookieDomainUrl('/'),
```

Comment:
This is not good. You are using `window.location.hostname` for redirectUri. That information is only "{domain_name}.{TLD}" => "github.com". This means that if you have site on https, if you setup only hostname as your `redirectUri`, you will be redirected to http version of the site.

Reply:
ack good point, this time around missed the distinction that we were using hostname in just the first domain, origin in the rest. Fixing.